client/me/billing-history/billing-history-table.jsx

```diff
@@ -37,12 +37,12 @@ class BillingHistoryTable extends React.Component {
 		const { translate } = this.props;
 
 		if ( this.props.sendingBillingReceiptEmail( receiptId ) ) {
-			return translate( 'Emailing Receipt…' );
+			return translate( 'Emailing receipt…' );
 		}
 
 		return (
 			<a href="#" onClick={ this.getEmailReceiptLinkClickHandler( receiptId ) }>
-				{ translate( 'Email Receipt' ) }
+				{ translate( 'Email receipt' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Email receipts' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)